drivers/src/hand_off.rs

```diff
@@ -427,6 +427,26 @@ impl FirmwareHandoffTable {
         None
     }
 
+    pub fn try_into_ref() -> Option<&'static FirmwareHandoffTable> {
+        let slice = unsafe {
```

Comment:
Assuming that FirmwareHandoffTable implements FromBytes and FHT_ORG is aligned properly, this is adequate:

```rs
let fht = unsafe { &*(FHT_ORG as *const FirmwareHandoffTable) };
```

We could make things a bit safer with something like this:

```rs
unsafe fn ref_from_addr<T: FromBytes>(addr: u32) -> &'static T {
  // LTO should be able to optimize this out to maintain panic_is_missing
  assert!(addr % core::mem::align_of<T>() == 0);
  assert!(core::mem::size_of::<u32>() == core::mem::size_of::<*const T>());
  &*(addr as *const T)
}
```

```rs
let fht = ref_from_addr::<FirmwareHandoffTable>(FHT_ORG);
```